@TheDiemer The build is failing with these errors:

```
pkg/dump/dump.go:43: File is not `gofmt`-ed with `-s` (gofmt)
// Simple functions exactly the same as Pretty except it uses jq's -c option to condense the 
cmd/uhc/get/cmd.go:33:10: struct of size 64 bytes could be of size 56 bytes (maligned)
var args struct {
         ^
```

To address the first issue you need to run `make fmt`.

To address the second issue you need to re-organize the fiedls of the `args` struct. Just put all the boolean fields together at the end:

```go
var args struct {
	parameter []string
	header    []string
	debug     bool
	single    bool
}
```

--- a/cmd/uhc/get/cmd.go
+++ b/cmd/uhc/get/cmd.go
@@ -31,9 +31,10 @@ import (
 )
 
 var args struct {
-	debug     bool
 	parameter []string
 	header    []string
+	debug     bool
+	single    bool
 }
 
 var Cmd = &cobra.Command{
@@ -68,6 +69,12 @@ func init() {
 			"followed by an optional equals sign and then the value of the "+
 			"header. Can be used multiple times to specify multiple headers "+
 			"or multiple values for the same header.",
+	)
+	flags.BoolVar(
+		&args.single,
+		"single",
+		false,
+		"Return the output as a single line.",
 	)
 }
 
@@ -159,9 +166,17 @@ func run(cmd *cobra.Command, argv []string) {
 	status := response.Status()
 	body := response.Bytes()
 	if status < 400 {
-		err = dump.Pretty(os.Stdout, body)
+		if args.single {
+			err = dump.Simple(os.Stdout, body)
+		} else {
+			err = dump.Pretty(os.Stdout, body)
+		}
 	} else {
-		err = dump.Pretty(os.Stderr, body)
+		if args.single {
+			err = dump.Simple(os.Stderr, body)
+		} else {
+			err = dump.Pretty(os.Stderr, body)
+		}
 	}
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Can't print body: %v\n", err)

--- a/pkg/dump/dump.go
+++ b/pkg/dump/dump.go
@@ -40,6 +40,21 @@ func Pretty(stream io.Writer, body []byte) error {
 	return dumpJSON(stream, data)
 }
 
+// Simple functions exactly the same as Pretty except it uses jq's -c option to condense the
+// output to a single line, intended to be used with other resources that require single line
+// output.
+func Simple(stream io.Writer, body []byte) error {
+	var data map[string]interface{}
+	err := json.Unmarshal(body, &data)
+	if err != nil {
+		return dumpBytes(stream, body)
+	}
+	if haveJQ() {
+		return dumpCondensedJQ(stream, body)
+	}
+	return dumpJSON(stream, data)
+}
+
 func dumpBytes(stream io.Writer, data []byte) error {
 	_, err := stream.Write(data)
 	if err != nil {
@@ -52,6 +67,15 @@ func dumpBytes(stream io.Writer, data []byte) error {
 func dumpJQ(stream io.Writer, data []byte) error {
 	// #nosec 204
 	jq := exec.Command("jq", ".")
+	jq.Stdin = bytes.NewReader(data)
+	jq.Stdout = stream
+	jq.Stderr = os.Stderr
+	return jq.Run()
+}
+
+func dumpCondensedJQ(stream io.Writer, data []byte) error {
+	// #nosec 204
+	jq := exec.Command("jq", "-c", ".")
 	jq.Stdin = bytes.NewReader(data)
 	jq.Stdout = stream
 	jq.Stderr = os.Stderr

